phpstan.neon

```diff
@@ -0,0 +1,14 @@
+parameters:
+  paths:
+    - src
+    - tests
+  level: 0
+  customRulesetUsed: true
+  ignoreErrors:
+      - '#\Drupal calls should be avoided in classes, use dependency injection instead#'
+      - '#Plugin definitions cannot be altered.#'
```

Comment:
If plugin definitions cannot be altered and we actually do it we may run into problems on runtime.

Reply:
Line removed